src/Truck.java

```diff
@@ -0,0 +1,14 @@
+public class Truck extends Transport {
+
+    public Truck(String modelName, int wheelsCount) {
+        super(modelName, wheelsCount);
+    }
+
+    public void checkEngine() {
+        System.out.println("Проверяем двигатель");
```

Comment:
```suggestion
        System.out.println("Проверяем двигатель грузовика");
```
реализация метода должна показывать полиморфизм - проверить двигатель у машины и у грузовика - это разные вещи, а у тебя одно и то же